backend/box/box.go

```diff
@@ -384,16 +390,25 @@ func NewFs(name, root string, m configmap.Mapper) (fs.Fs, error) {
 	}
 
 	root = parsePath(root)
-	oAuthClient, ts, err := oauthutil.NewClient(name, m, oauthConfig)
-	if err != nil {
-		return nil, errors.Wrap(err, "failed to configure Box")
+
+	boxAccessToken, boxAccessTokenOk := m.Get("box_primary_access_token")
+
+	client := &http.Client{}
+	var ts *oauthutil.TokenSource
+
+	// If not using an accessToken, create an oauth client and tokensource
+	if !boxAccessTokenOk || boxAccessToken == "" {
```

Comment:
I would have thought `f.opt.AccessToken == ""` would be good enough here? A blank access token will never be valid will it? You can lose the `m.Get` above as well if so.